[WIP] Adding access considerations and support to community handbook

aleesteele opened the pull request:
### Summary

Currently drafting a list of access and inclusion-related support that we provide for online events for the community handbook, related to #3489 

### Acknowledging contributors

<!-- Please select the correct box -->

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file: <!-- replace this text with the GitHub IDs of any new contributors -->

aleesteele commented:
Priority for week of 29 January - 2 February

LizHareDogs reviewed (commented):
These pages look great!

Consider removing the character limit for alt-texts. Being concise is already mentioned once and the length should depend on the complexity of the content. 

Consider removing the statement that it helps search engines. We dno't need to justify accessibility practices by saying they have other uses.

Consider adding a suggestion like:
THe alt-text should convey the main point of the image. This is often an explanation of why the image was included or what the author wants the reader to notice.

aleesteele commented:
Thanks so much @LizHareDogs for your review! I added a few edits – noting specifically the comments you mentioned. For the one about justifying use for searchability, I've added a qualifier, but it might be a worthwhile "learning" point to keep in the text. What do you think?

LizHareDogs commented:
The problem is that some people use the alt text as a list of search term hits rather than to convey the *meaning* of the image. The proper use of alt text is for access, not SEO.

> On Feb 22, 2024, at 7:53 AM, Anne Lee Steele ***@***.***> wrote:
> 
> 
> Thanks so much @LizHareDogs <https://github.com/LizHareDogs> for your review! I added a few edits – noting specifically the comments you mentioned. For the one about justifying use for searchability, I've added a qualifier, but it might be a worthwhile "learning" point to keep in the text. What do you think?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/the-turing-way/the-turing-way/pull/3491#issuecomment-1959398351>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMVDYUGBQESZEIARSMJCKP3YU45TTAVCNFSM6AAAAABCB7QJOSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSNJZGM4TQMZVGE>.
> You are receiving this because you were mentioned.
> 

Liz Hare
***@***.***
Quantitative Geneticist, Dog Genetics LLC
Assistant Director, Data Management and Open Science, ManyDogs Project <https://manydogsproject.github.io/>

Co-Lead, Accessibility Working Group, The Turing Way <https://the-turing-way.netlify.app/index.html>

aleesteele commented:
Thanks Liz, that's helpful to know. I can see what you mean, as we have also discussed how alt text is frequently isued for jokes or humor. Such as: 
- https://www.wsj.com/articles/misuse-of-twitters-alt-text-feature-draws-criticism-from-accessibility-advocates-11657879200
- https://twitter.com/p0ppyfield/status/1535221628338069505?lang=en
- https://twitter.com/SarcasmWelcome/status/1653238678464897024

I'm going to make an edit to account for both of these!

aleesteele commented:
Alright - I made an edit using the following language instead: "While Alt text has been used in the context of humour or SEO visibility, it is important to remember that this is _not_ the purpose of Alt text more broadly, as a feature that was originally designed to enable blind and low-vision people to understand the contents of images through text."

aleesteele commented on book/website/community-handbook/accessibility/alt-text.md:
```diff
@@ -0,0 +1,124 @@
+All our chapters are written in Markdown files.
+Therefore, using Markdown syntax to include a figure in a Markdown file will work fine, for example, `![](../../figures/file-collection.*)`, where the relative path of the image file is provided inside the round brackets '()'.
+
+**However**, this formatting does not allow images to be responsive to screen sizes, making them inaccessible to read on small screens and smartphones.
```
```suggestion
However, this formatting does not allow images to be responsive to screen sizes, making them inaccessible to read on small screens and smartphones.
```

BrainonSilicon reviewed (approved):
Looks good to me! 

I made a change to the tip vs. heading as discussed in the AWG call.
